en/getting_started/installation.md

```diff
@@ -38,7 +38,7 @@ To install the MAVLink toolchain:
 1. Install the required packages:
 
    ```
-   python3 -m pip install -r pymavlink/requirements.txt
+   python3 -m pip install -r mavlink/pymavlink/requirements.txt
```

Comment:
Thanks. How about instead in the first step we navigate into the directory. My reasoning is that the next steps will all assume you are inside that root?